### ISSUE TITLE
Nur in der Legende anzeigen funktioniert nicht

### ISSUE THREAD

**Issue by Bernd9000** [ioBroker/ioBroker.echarts#251]
**Describe the bug**  
Diagramm soll nicht angezeigt werden.
Dazu wird unter einer Datenquelle "Nur in der Legende anzeigen" angehackt.
Ergebnis: Diagramm wird trotzdem angezeigt

**To Reproduce**  
1. "Nur in der Legende anzeigen" anhacken
2. Diagramm anschauen

**Expected behavior**  
Diagramm sollte nicht angezeigt werden wenn Option ausgewählt wurde

**Screenshots & Logfiles**  
![Unbenannt](https://user-images.githubusercontent.com/77005685/217666827-758edcb7-fc13-4bff-9462-0b6561b5244a.jpg)

**Versions:**  
 - Adapter version: 1.3.4
 - JS-Controller version: 4.0.2.4
 - Node version: v16.19.0
 - Operating system: Windows 11

**Additional context**

**Comment by GermanBluefox** [comment]
Fixed in 1.3.5

**Comment by MichaelSchaaf999** [comment]
Danke sehr, Update machen wir dann, wenn es angeboten wird, noch steht da 1.3.4 im Repo.

**Comment by Bernd9000** [comment]
@MichaelSchaaf999 
Das kann ich dir nicht sagen, aber wird bestimmt bald kommen.
Ich hab die Version 1.3.5 im Expertenmodus über "Benutzerdefinierte URL" installiert.

**Comment by MichaelSchaaf999** [comment]
ja, aber sorry: In Github existiert anscheinend keine V 1.3.5, dafür aber eine 1.4.0. 
Wassn'nu?
Und: Wo findet man denn dann die Installations-URL?

**Comment by sigi2345** [comment]
> ja, aber sorry: In Github existiert anscheinend keine V 1.3.5, dafür aber eine 1.4.0. Wassn'nu? Und: Wo findet man denn dann die Installations-URL?

![Screenshot (5347)](https://user-images.githubusercontent.com/41079106/218673455-0186e0a9-3ff5-45f0-8ec3-e775ee47b1ca.jpg)

**Comment by MichaelSchaaf999** [comment]
Vielen Dank!!! Hab ich ja noch niiiie gemacht.
Lief dann aber. Thanks!
![image](https://user-images.githubusercontent.com/53481090/218682327-f88f7398-9f2a-432b-a95d-dbd3f81312a4.png)

![image](https://user-images.githubusercontent.com/53481090/218682837-0f65f864-6f4c-40fd-8b8b-5fdcab2b3fcb.png)

ABER wenn ich dann die Werte für die Legende einschalte:

![image](https://user-images.githubusercontent.com/53481090/218683049-5751d8b5-9f76-491e-bb33-f714af8ad63f.png)

![image](https://user-images.githubusercontent.com/53481090/218683160-e6805724-2c13-45bc-b664-14f77b10704e.png)

Dann:

![image](https://user-images.githubusercontent.com/53481090/218683263-852d8ef0-b68d-4b18-a28a-45e65b34d76e.png)

Und die selektierte Menüoption hat hier übrigens auch die Farbe des Hintergrunds und ist somit nicht lesbar. Das war aber auch schon in V 1.3.x so.

**Comment by MichaelSchaaf999** [comment]
Legende ist noch anfällig:
Hier andere Beispiele für "ohne Werte-Anzeigen"
![image](https://user-images.githubusercontent.com/53481090/218952285-1cf44862-29b7-4204-b3fe-81be3563d0a8.png)
Hier mit-Anzeigen
![image](https://user-images.githubusercontent.com/53481090/218952479-91fefcd8-94f2-41cb-b564-6d04b0d9853a.png)

In diesem Beispiel wird auch nur die Legende für eine Kurve angezeigt
![image](https://user-images.githubusercontent.com/53481090/218952821-280a7a83-4565-482b-a69f-4afa424a862b.png)

In anderen Graphen funktioniert die Legende hingegen, ob mit oder ohne Werte-anzeigen
![image](https://user-images.githubusercontent.com/53481090/218953069-b32e921a-4417-4308-955c-0ed614506945.png)
